x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/edit_exception_flyout/index.tsx

```diff
@@ -38,6 +38,7 @@ import type {
 import { getExceptionBuilderComponentLazy } from '@kbn/lists-plugin/public';
```

Comment:
Just to understand 😄, I see we have two `flyout` one for Add, and for Edit, they look to me very similar am I missing any differences?

Reply:
With edit you can't add multiple exception items, just edit the single exception item (it's considered multiple items when a user selects the `OR` logic). Also for edit, user can't change the selection of adding to a rule or list, we just display for the user a read only section.

Reply:
Thanks for the clarification, I believe we can merge both of them in one and just turn on/off the edit section in case of the Edit mode. However, I see it will be a big change so we can postpone it :)